Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
-name = "cargo_metadata"
-version = "0.2.3"
 authors = ["Oliver Schneider <git-spam-no-reply9815368754983@oli-obk.de>"]
-repository = "https://github.com/oli-obk/cargo_metadata"
 description = "structured access to the output of `cargo metadata`"
 license = "MIT"
-
+name = "cargo_metadata"
```

Comment:
What happened here? These changes look rather unrelated. I can understand ordering the dependencies by name, but the package entries should stay in the idiomatic order.